site/en/docs/privacy-sandbox/events/index.md

```diff
@@ -16,9 +16,16 @@ already taken place.
 
 ## Future events
 
-There are no new events confirmed right now. 
+### Privacy Sandbox Office Hours: Ask Me Anything {: #oh05}
+**Privacy Sandbox Developer Office Hours #5**
 
-Check back here for updates, or subscribe to the following mailing lists to be notified of events:
+**Session A (APAC time-friendly)** 
+* Monday, September 19, 8 – 9:00pm PDT • Tuesday 20 September, 8:30am IST / 11am SIN / 1pm AEST / 3pm NZST
+* Video call: [meet.google.com/wqr-ubfx-tze]([url](https://meet.google.com/wqr-ubfx-tze))
+* Or dial: ‪(US) +1 503-755-4475‬ PIN: ‪885 701 945‬#
+* More phone numbers: [tel.meet/wqr-ubfx-tze?pin=6856114658371]([url](https://tel.meet/wqr-ubfx-tze?pin=6856114658371))
+
```

Comment:
Previously, we only published Meet information in a public Chromium Google Doc, and not on the events page: https://docs.google.com/document/d/14GNexALd7dLBJe-MdLFuAjiZI3EhfNvk8zTs5Ct6nnM/edit#heading=h.yqt25rpr2jgl

I'll share the doc with you momentarily.  I think there's a procedure around why this is the case, but I haven't found it yet. Perhaps it's an old procedure that no longer should be followed! Will follow up when I know more.